Sources/TensorFlow/Layers/Pooling.swift

```diff
@@ -424,3 +424,51 @@ public struct GlobalMaxPool3D<Scalar: TensorFlowFloatingPoint>: ParameterlessLay
     return input.max(squeezingAxes: [1, 2, 3])
   }
 }
+
+/// A fractional max pooling layer.
+///
+/// Note: `FractionalMaxPool` does not have an XLA implementation, and thus may have performance implications.
+@frozen
+public struct FractionalMaxPool<Scalar: TensorFlowFloatingPoint>: ParameterlessLayer {
```

Comment:
It seems `FractionalMaxPool` supports only 2D inputs.
Shouldn't it be `FractionalMaxPool2D` like `MaxPool2D`?

Reply:
Done